src/components/ListItem.jsx

```diff
@@ -1,11 +1,14 @@
 import './ListItem.css';
 import { useCallback, useEffect, useState } from 'react';
-import { updateItem } from '../api/firebase';
+import { updateItem, deleteItem } from '../api/firebase';
 import { getFutureDate } from '../utils';
+import Button from './Button';
+import DeleteItemModal from './DeleteItemModal';
 
 export function ListItem({ listToken, item, itemId }) {
 	const { name, dateLastPurchased, checked } = item;
 	const [isPurchased, setIsPurchased] = useState(checked);
+	const [modalStatus, setModalStatus] = useState(false);
```

Comment:
nit: boolean variables are often named with a name that starts with 'is' (sometimes has/can/should). I think this convention became popular with Sun Microsystems and its Java core packages. But it's generally good practice and may be required on your future teams/style guides